geoips/interfaces/base.py

```diff
@@ -170,18 +170,46 @@ def validate(self, plugin, validator_id=None):
         except (KeyError, AttributeError, TypeError):
             pass
 
+        # This is a temporary fix as we transition to using one top-level schema per
+        # interface. As we transition, add more exceptions here.
+        if not validator_id and plugin["interface"] == "sectors":
+            if plugin["family"] == "generated":
+                validator_id = "sectors.generated"
+            else:
+                validator_id = "sectors.static"
+
         if not validator_id:
             self.validators["bases.top"].validate(plugin)
             validator_id = f"{plugin['interface']}.{plugin['family']}"
+
         try:
             validator = self.validators[validator_id]
-        except KeyError:
+        except KeyError as err:
             raise ValidationError(
                 f"No validator found for '{validator_id}'"
                 f"\nfrom plugin '{plugin['name']}'"
                 f"\nin interface '{plugin['interface']}'"
-            )
+            ) from err
+
+        # This turned out to be too big of a change for now.
+        # We should consider how to implement something like this while still being able
+        # to test the preceeding error in our unit tests. Currently, the error ourput by
+        # `valicator.validate(plugin)` is being used for testing in the unit tests.
+        #
+        # See issue #303
```

Comment:
I created issue #303 to deal with this, but would like to leave this as an example for use in #303.